File: Auth_Services/Auth_Services/DTOs/TeacherScheduleDetailDTO.cs
namespace Auth_Services.DTOs
{
    public class TeacherScheduleDetailDTO
    {
        public DateTime DateTime { get; set; }
        public int TurmaId { get; set; }
        public string TurmaName { get; set; }
        public int ModuleId { get; set; }
        public string ModuleName { get; set; }
        public int TotalDuration { get; set; }
        public int HoursCompleted { get; set; }
        public int SalaId { get; set; }
        public string SalaNome { get; set; }
        public int HasPc { get; set; }
        public int HasOficina { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/example/nts_app/network/TeacherScheduleDetailDTO.kt
package com.example.nts_app.network

import com.google.gson.annotations.SerializedName

data class TeacherScheduleDetailDTO(
    val dateTime: String,
    val turmaId: Int,
    val turmaName: String,
    val moduleId: Int,
    val moduleName: String,
    val totalDuration: Int,
    val hoursCompleted: Int,
    val salaId: Int,
    val salaNome: String,
    val hasPc: Int,
    val hasOficina: Int
)